packages/docs/pages/flex.tsx

```diff
@@ -24,153 +24,225 @@ const ExampleBox: React.FC<{ vertical?: boolean }> = ({ children, vertical }) =>
-            </CodePreview>
-          </Panel>
-          <Panel header="Justify content">
-            <Text>
-              Flex container's can space their element with uniform spacing in-between each flex item using the{' '}
-              <Code primary>justifyContent</Code> prop.
-            </Text>
+  return (
+    <>
+      <H1>Flex</H1>
 
-            <CodePreview scope={{ ExampleBox }}>
-              {/* jsx-to-string:start */}
-              <Flex justifyContent="space-between">
-                <FlexItem>
-                  <ExampleBox>Item 1</ExampleBox>
-                </FlexItem>
-                <FlexItem>
-                  <ExampleBox>Item 2</ExampleBox>
-                </FlexItem>
-                <FlexItem>
-                  <ExampleBox>Item 3</ExampleBox>
-                </FlexItem>
-                <FlexItem>
-                  <ExampleBox>Item 4</ExampleBox>
-                </FlexItem>
-              </Flex>
-              {/* jsx-to-string:end */}
-            </CodePreview>
-          </Panel>
-          <Panel header="Column layout">
-            <Text>
-              One way of creating a column based layout is using a combination of the <Code primary>flexWrap</Code> prop
-              on the flex container with an additional <Code primary>flexBasis</Code> prop on the flex items.
-            </Text>
+      <Panel header="Overview" headerId="overview">
+        <Text>
+          A <Code primary>Flex</Code> container used for customizable layouts.
+        </Text>
 
-            <CodePreview scope={{ ExampleBox }}>
-              {/* jsx-to-string:start */}
-              <Flex flexWrap="wrap">
-                <FlexItem flexBasis="100%">
-                  <ExampleBox vertical>Item 1</ExampleBox>
-                </FlexItem>
-                <FlexItem flexBasis="100%">
-                  <ExampleBox vertical>Item 2</ExampleBox>
-                </FlexItem>
-                <FlexItem flexBasis="100%">
-                  <ExampleBox vertical>Item 3</ExampleBox>
-                </FlexItem>
-                <FlexItem flexBasis="100%">
-                  <ExampleBox vertical>Item 4</ExampleBox>
-                </FlexItem>
-              </Flex>
-              {/* jsx-to-string:end */}
-            </CodePreview>
-          </Panel>
-          <Panel header="Sort content">
-            <Text>
-              Using the <Code primary>flexOrder</Code> prop you can reorganize content based on the size generated by
-              setting the <Code primary>flexGrow</Code> prop. <Code primary>flexGrow</Code> allows the flex items to
-              grow relative to the flex container's width.
-            </Text>
+        <Text bold>When to use:</Text>
+        <List>
+          <List.Item>Want to lay a collection of items out in one direction or another.</List.Item>
+          <List.Item>
+            Want to control the dimensions of the items in that one dimension, or control the spacing between items.
```

Comment:
I pulled these from [here](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Typical_Use_Cases_of_Flexbox). I agree with it, since you can definitely control the dimension by setting how much space each element should take in an unknown container.